### ISSUE TITLE
Terrible color contrast in some places

### ISSUE THREAD

**Issue by auroraanna** [jaedb/Iris#966]
### Iris version

3.69.3

### Operating system(s) affected

- [X] Windows
- [X] MacOS
- [X] iOS
- [X] Android
- [X] Linux
- [X] Other

### Browser(s) affected

- [X] Firefox
- [X] Chrome
- [X] Edge
- [X] Other

### What happened?

The left-hand sidebar categorises things in to "DISCOVER" and "MY MUSIC". However [they both have terrible color contrast in the dark theme](https://webaim.org/resources/contrastchecker/?fcolor=494949&bcolor=121212).

I've seen bad color contrast in other places in this app too. Please evaluate the color contrast of the entire app.

**Comment by jaedb** [comment]
Thanks for your feedback! I have marked this as Accepting Votes for now as it would be worthwhile gauging how other users find the contrast before investing effort into revisiting the entire UI.

**Comment by auroraanna** [comment]
Your application is failing all Web Content Accessibility Guidelines. These guidelines apply to your application because it is a web app. Accessibility should not be chosen for all users by a few GitHub users. The default for all users should be good accessibility. If users do not like it there could be settings to adjust the color contrast to something with less contrast.

You might have already scared away users of using your application by not having proper color contrast. It could be that they never complained to you.
